go.mod

```
@@ -4,6 +4,8 @@ go 1.23.2
 
 toolchain go1.23.4
 
+replace go.opencensus.io => github.com/census-instrumentation/opencensus-go v0.24.0
```

can we scope this down?
```suggestion
replace go.opencensus.io v0.24.0 => github.com/census-instrumentation/opencensus-go v0.24.0
```